src/MagnumPlugins/BasisImageConverter/CMakeLists.txt

```diff
@@ -64,6 +64,10 @@ target_include_directories(BasisImageConverter PUBLIC
     # Turns #include "../zstd/zstd.h" into an actual external zstd.h include.
     # See the README in that directory for details.
     ${PROJECT_SOURCE_DIR}/src/external/basis-zstd-include-uncrapifier/put-this-on-include-path)
+# Adds empty definitions for unused image loading library functions.
+# See the README in that directory for details.
+target_sources(BasisImageConverter PRIVATE
+    ${PROJECT_SOURCE_DIR}/src/external/basis-image-loading-stubs/stubs.cpp)
```

Comment:
This is great, thanks.

I'd just put this together with the above zstd workaround, let's say a dir named `basis-uncrapifier` containing both, since both are used together anyway.